provisioning/device/samples/register_symkey.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var iotHubTransport = require('azure-iot-device-mqtt').Mqtt;
+var Client = require('azure-iot-device').Client;
+var Message = require('azure-iot-device').Message;
+
+var ProvisioningTransport = require('azure-iot-provisioning-device-http').Http;
```

Comment:
>Http [](start = 74, length = 4)

nit: would rather have MQTT by default I think.

Reply:
Done.

---
In reply to: [231344673](https://github.com/Azure/azure-iot-sdk-node/pull/418#discussion_r231344673) [](ancestors = 231344673)